apps/evm/src/libs/analytics/useAnalytics.ts

```diff
@@ -125,7 +132,37 @@ export type AnalyticEventProps<TEventName extends AnalyticEventName> =
                                     }
                                   : TEventName extends 'Vote cast'
                                     ? { proposalId: number; voteType: string }
-                                    : undefined;
+                                    : TEventName extends 'Position import initiated'
+                                      ? {
+                                          fromProtocol: ImportableProtocol;
+                                          fromTokenSymbol: string;
+                                          fromTokenAmountTokens: number;
+                                          fromTokenAmountDollars: number;
+                                          fromTokenApyPercentage: number;
+                                          toVTokenAddress: Address;
+                                          toTokenApyPercentage: number;
+                                        }
+                                      : TEventName extends 'Position import failed'
+                                        ? {
+                                            fromProtocol: ImportableProtocol;
+                                            fromTokenSymbol: string;
+                                            fromTokenAmountTokens: number;
+                                            fromTokenAmountDollars: number;
+                                            fromTokenApyPercentage: number;
+                                            toVTokenAddress: Address;
+                                            toTokenApyPercentage: number;
+                                          }
+                                        : TEventName extends 'Position imported'
+                                          ? {
+                                              fromProtocol: ImportableProtocol;
+                                              fromTokenSymbol: string;
+                                              fromTokenAmountTokens: number;
+                                              fromTokenAmountDollars: number;
+                                              fromTokenApyPercentage: number;
+                                              toVTokenAddress: Address;
+                                              toTokenApyPercentage: number;
+                                            }
+                                          : undefined;
```

Comment:
nit: this also applies to other conditional types here, they can be merged
```suggestion
                                    : TEventName extends
                                          | 'Position import initiated'
                                          | 'Position import failed'
                                          | 'Position imported'
                                      ? {
                                          fromProtocol: ImportableProtocol;
                                          fromTokenSymbol: string;
                                          fromTokenAmountTokens: number;
                                          fromTokenAmountDollars: number;
                                          fromTokenApyPercentage: number;
                                          toVTokenAddress: Address;
                                          toTokenApyPercentage: number;
                                        }
```